main.tf

```diff
@@ -11,34 +11,28 @@ data "aws_route53_zone" "parent_zone" {
   name    = var.parent_zone_name
 }
 
-data "template_file" "zone_name" {
-  count    = local.enabled
-  template = replace(var.zone_name, "$$", "$")
-
-  vars = {
-    namespace        = module.this.namespace
-    environment      = module.this.environment
-    name             = module.this.name
-    stage            = module.this.stage
-    id               = module.this.id
-    attributes       = join(module.this.delimiter, module.this.attributes)
-    parent_zone_name = coalesce(join("", data.aws_route53_zone.parent_zone.*.name), var.parent_zone_name)
-    region           = data.aws_region.default.name
-  }
-}
-
 resource "aws_route53_zone" "default" {
   count = local.enabled
   name  = join("", data.template_file.zone_name.*.rendered)
```

Comment:
@kevcube I think you updated the wrong resource in your updates. See the following error from our tests: 

```
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121: Error: Reference to undeclared resource
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121: 
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121:   on ../../main.tf line 16, in resource "aws_route[53](https://github.com/cloudposse/actions/runs/5223731548?check_suite_focus=true#step:9:53)_zone" "default":
TestExamplesComplete 2022-02-16T21:[57](https://github.com/cloudposse/actions/runs/5223731548?check_suite_focus=true#step:9:57):44Z command.go:121:   16:   name  = join("", data.template_file.zone_name.*.rendered)
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121: 
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121: A data resource "template_file" "zone_name" has not been declared in
TestExamplesComplete 2022-02-16T21:57:44Z command.go:121: module.domain.
```

Reply:
@Gowiem you're right! fixed